libs/env/index.js

```diff
@@ -1,12 +1,13 @@
 import fs from "node:fs";
 import path from "node:path";
-import { cwd } from "node:process";
+import { fileURLToPath } from "node:url";
 
 import dotenv from "dotenv";
 
 import { VALID_FLAW_CHECKS } from "../constants/index.js";
 
-export const ROOT = path.join(cwd(), ".");
+const currentDir = path.dirname(fileURLToPath(import.meta.url));
+export const ROOT = path.join(currentDir, "..", "..");
```

Comment:
should be a revert of https://github.com/mdn/yari/commit/d3f7ec15aeb45145849683116ccda7455897ffa1#diff-309d7c0018f34c8bb48de097e4b860c64e2a2bb987dbce04f88f10a2bb7cb559L9

Reply:
I think `fileURLToPath(new URL("../..", import.meta.url))` should work (see other usages of `import.meta.url` in the repo.

Reply:
Tested, and applied

Reply:
Webpack actually doesn't like us consolidating this into one line, which is why I opted for `cwd()` I think.  Webpack attempts to resolve `../..` when creating it as a URL.

Reply:
Should we revert 8666343

Reply:
Yeah, we should revert that commit -- perhaps we should also add a comment above the lines explaining that we need to do it this way to prevent Webpack from attempting to resolve "../.." as an import?